src/app/components/content-page/content-page.component.html

```diff
@@ -1,110 +1,24 @@
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-  </div>
-
-  <div class="d-flex software-box-outer">
-    <div class="p-2 card  photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-  </div>
-
-  <div class="d-flex software-box-outer">
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-  </div>
-
-  <div class="d-flex software-box-outer">
-    <div class="p-2 card  photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-  </div>
-
-  <div class="d-flex software-box-outer">
-    <div class="p-2 card  photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-  </div>
-
-  <div class="d-flex software-box-outer">
-    <div class="p-2 card  photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-  </div>
-
-  <div class="d-flex software-box-outer">
-    <div class="p-2 card  photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
-    <div class="p-2 card photo-box-marign">
-      <img class="card-img" [src]="catAvatar3" alt="Cat image">
-    </div>
+<main class="container px-1">
+  <div *ngFor="let content of tempContent">
+    <img class="item-img w-100 h-100 p-1" [src]="content" alt="Picture">
```

Comment:
Shouldn't [src] be src only? Why is there square brackets around it?

Reply:
This is apparently how it is done, see example here https://stackoverflow.com/questions/40797925/how-to-bind-img-src-in-angular-2-in-ngfor

Reply:
Cool that make sense. 👍